appsettings.json

```diff
@@ -5,5 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "MongoDbSettings": {
+    "ConnectionString": "mongodb+srv://aliakseidziatlau:Alexey200616may@officeservicecluster.0dsu1.mongodb.net/?retryWrites=true&w=majority&appName=OfficeServiceCluster",
```

Comment:
usually such data is not sent to the repository, it can be saved in user secrets.